internal/checker/printer.go

```diff
@@ -79,6 +79,99 @@ func (c *Checker) typeToStringEx(t *Type, enclosingDeclaration *ast.Node, flags
 	return p.string()
 }
 
+func (c *Checker) GetQuickInfoAtLocation(node *ast.Node) string {
+	symbol := c.GetSymbolAtLocation(node)
+	if symbol != nil && symbol.Flags&ast.SymbolFlagsAlias != 0 {
+		symbol = c.resolveAlias(symbol)
```

Comment:
Maybe this is just extra, but I think that if you remembered if this path were hit, you could get `(alias)` too, like tsserver:

![image](https://github.com/user-attachments/assets/5a1a0d4b-b858-4dfa-a250-ec6ac68632e5)

Versus the current PR's:

![image](https://github.com/user-attachments/assets/baa30b21-c750-460f-9a13-70e23c59a00a)

Reply:
Yup, will do.